components/addContact/index.test.jsx

```diff
@@ -126,28 +125,6 @@ describe("handleSubmit", () => {
     expect(alertError).toHaveBeenCalledWith(EXISTING_WEBID_ERROR_MESSAGE);
     expect(setDirtyForm).toHaveBeenCalledWith(true);
   });
-  test("it alerts the user and exits if the webid doesn't have a name", async () => {
```

Comment:
Should this test be changed to show that `alertSuccess` is called with the webid, instead?